Sources/MapboxDirectionsCLI/ProcessCommand.swift

```diff
@@ -0,0 +1,128 @@
+
+import Foundation
+import MapboxDirections
+import SwiftCLI
+
+
+private let BogusCredentials = DirectionsCredentials(accessToken: "pk.feedCafeDadeDeadBeef-BadeBede.FadeCafeDadeDeed-BadeBede")
+
+class ProcessCommand<ResponceType : Codable, OptionsType : DirectionsOptions > : Command {
+    
+    // MARK: - Parameters
+    
+    var name = "process"
+    
+    @Key("-i", "--input", description: "Filepath to the input JSON.")
+    var inputPath: String?
+    
+    @Key("-c", "--config", description: "Filepath to the JSON, containing serialized Options data.")
+    var configPath: String?
+    
+    @Key("-o", "--output", description: "[Optional] Output filepath to save the conversion result. If no filepath provided - will output to the shell.")
+    var outputPath: String?
+    
+    @Flag("-t", "--text", description: "Output as a plain text. Does not work together with '--json' flag")
+    var textOutput: Bool
+    
+    @Flag("-j", "--json", description: "Output as a JSON text. Does not work together with '--text' flag")
+    var jsonOutput: Bool
```

Comment:
Consider replacing these mutually exclusive flags with a single `--format` option that takes either `text` or `json` as an argument.

Reply:
Done